client/signup/reskinned-processing-screen/index.jsx

```diff
@@ -64,27 +81,41 @@ export default function ReskinnedProcessingScreen( props ) {
 			<h1 className="reskinned-processing-screen__progress-step">
 				{ steps.current[ currentStep ] }
 			</h1>
-			<div
-				className="reskinned-processing-screen__progress-bar"
-				style={ {
-					'--progress': ! hasStarted ? /* initial 10% progress */ 0.1 : progress,
-				} }
-			/>
-			<p className="reskinned-processing-screen__progress-numbered-steps">
-				{
-					// translators: these are progress steps. Eg: step 1 of 4.
-					sprintf( __( 'Step %(currentStep)d of %(totalSteps)d' ), {
-						currentStep: currentStep + 1,
-						totalSteps,
-					} )
-				}
-			</p>
+			{ isSetupSiteFlow && (
```

Comment:
In your animation p1634277774338900/1634086694.194100-slack-C029SB8JT8S your showing the ellipses during headstart. Should this logic instead be 
```suggestion
			{ ( isSetupSiteFlow || flowName === 'setup-site' )&& (
```

Another reason I thought the `isSetupSiteFlow` prop name was confusing 😃

Reply:
oh yes! good catch, I confused myself with that 😅